src/native-common/FrontLayerViewManager.tsx

```diff
@@ -34,6 +34,8 @@ const _styles = {
     }
 };
 
+const MAX_CACHED_POPUPS = 4;
```

Comment:
Ideally this would be controller by an app policy. Do we have a mechanism for apps to pass global settings down to ReactXP?

Reply:
No, and I don't think this should be controlled by app policy. There are many caches within React and React Native that are opaque to the app. Exposing all of these knobs is unnecessary and would just add complexity.